goal_src/jak1/levels/jungle/jungle-mirrors.gc

```diff
@@ -654,15 +654,15 @@
         (when (not (or (paused?) (-> (the-as periscope (-> self change-event-from 0)) aligned?)))
           (vector-reset! s5-0)
           (when *camera-read-analog*
-            (let ((f26-0 (analog-input
+            (let ((f26-0 (analog-input-horizontal-first
```

Comment:
add note that it was changed for the pc port

Reply:
Added a comment :fox_face: